### PR TITLE
removes outdated tvu_forward socket

### PR DESCRIPTION
**Pull request by behzadnouri**
#### Problem
Shreds are no longer sent to `tvu_forward`.

#### Summary of Changes
Removed outdated `tvu_forward` socket.

### CODE REVIEW

File: gossip/src/cluster_info.rs [hunk 1]
```diff
@@ -826,7 +826,7 @@ impl ClusterInfo {
                     }
                     let ip_addr = node.gossip().as_ref().map(SocketAddr::ip).ok();
                     Some(format!(
-                        "{:15} {:2}| {:5} | {:44} |{:^9}| {:5}|  {:5}| {:5}| {:5}| {:5}| {:5}| {:5}| {:5}| {}\n",
+                        "{:15} {:2}| {:5} | {:44} |{:^9}| {:5}|  {:5}| {:5}| {:5}| {:5}| {:5}| {:5}| {}\n",
```
Comment (steviez) [inline gossip/src/cluster_info.rs:829]: The table header lower down in this function would need to be adjusted too right ?

**Review by steviez — COMMENTED** [review]
Glancing at metrics, I still see some nodes reporting `shred_fetch_tvu_forwards.shred_count` on both testnet and mainnet. I haven't dug in much deeper yet, but these are definitely a minority of the cluster. Custom logic on their end ?

File: gossip/src/cluster_info.rs (hunk 1 above)
Comment (behzadnouri) [reply gossip/src/cluster_info.rs:829]: done.

**Comment by behzadnouri** [comment]
> Glancing at metrics, I still see some nodes reporting `shred_fetch_tvu_forwards.shred_count` on both testnet and mainnet. I haven't dug in much deeper yet, but these are definitely a minority of the cluster. Custom logic on their end ?

Jito shred-forwarders I think send to tvu-forward but they can just change their code and send it tvu.
As can be seen in this commit solana-client doesn't actually send any packets to tvu-forward.

**Review by steviez — APPROVED** [review]
> Jito shred-forwarders I think send to tvu-forward but they can just change their code and send it tvu.
As can be seen in this commit solana-client doesn't actually send any packets to tvu-forward.

Good call on it being jito shred forwarders. Change seems fine to me.

@buffalu - Heads up on this PR, tvu forward port going away and it looks like you all might be using it. This change would land in v1.17 branch, so should be enough time to update + communicate change to folks running jito

File: gossip/src/cluster_info.rs [hunk 2]
```diff
@@ -3224,92 +3206,6 @@ mod tests {
         ));
     }
 
-    #[test]
-    fn test_cluster_info_trace() {
```
Comment (steviez) [inline gossip/src/cluster_info.rs:3228]: You didn't feel like keeping this one up to date anymore 😅 ?

Comment (behzadnouri) [reply gossip/src/cluster_info.rs:3228]: :smile:

**Pull request merged**